README.md

```diff
@@ -137,6 +149,11 @@ helm install external-secrets \
 helm install adaptive adaptive/adaptive -f charts/adaptive/values_external_secret.yaml
 ```
 
+## About persistence and volumes
+
+- **monitoring** stack helm chart: by default Logs and Grafana data are not persisted. You should enable `grafana.enablePersistence=true` and set `grafana.storageClass` to an existing storage class name in target k8s cluster.
+- **adaptive** helm chart: it installs Prometheus which may require metrics data being persisted. By default `prometheus.server.persistentVolume.enabled=false`. When enabling peristence, i'll have to specify the used storage class name: `prometheus.server.storageClass`.
```

Comment:
`i'll have to` => `you will have to` 

you already the `you` in the bullet point above